packages/ui/src/types/index.ts

```diff
@@ -34,7 +33,7 @@ export enum ConfirmModalMessage {
 
 export type PreviewConfig = {
   signInExperience: SignInExperienceSettingsResponse;
-  language: LanguageKey;
+  language: string;
```

Comment:
Maybe `LangaugeTag`?

Reply:
nope technically it is loaded from a global post message response, can't guarantee it's type